src/Tools/Common/Commands/ProcessStatus.cs

```diff
@@ -3,30 +3,111 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.Diagnostics.NETCore.Client;
+using Microsoft.Tools.Common;
 using System;
 using System.CommandLine;
 using System.CommandLine.Invocation;
 using System.CommandLine.IO;
+using System.Collections.Generic;
 using System.Diagnostics;
 using System.Linq;
+using System.Runtime.InteropServices;
 using System.Text;
 using Process = System.Diagnostics.Process;
+using System.IO;
+using System.ComponentModel;
+using System.Threading.Tasks;
+using Microsoft.Diagnostics.Tools.Trace.CommandLine;
+using System.CommandLine.Binding;
 
 namespace Microsoft.Internal.Common.Commands
 {
     public class ProcessStatusCommandHandler
     {
-        public static Command ProcessStatusCommand(string description) =>
-            new Command(name: "ps", description)
+        public static Command ProcessStatusCommand() =>
+            new Command(name: "ps", 
+            description: "Lists the dotnet processes that traces can be collected")
             {
-                Handler = CommandHandler.Create<IConsole>(PrintProcessStatus)
+                HandlerDescriptor.FromDelegate((ProcessStatusDelegate)ProcessStatus).GetCommandHandler()
             };
-
+        delegate Task<int> ProcessStatusDelegate(IConsole console);
+        
+        //private static string formatLength()
         /// <summary>
-        /// Print the current list of available .NET core processes for diagnosis and their statuses
+        /// Print the current list of available .NET core processes for diagnosis, their statuses and the command line arguments that are passed to them.
         /// </summary>
-        public static void PrintProcessStatus(IConsole console)
+        private static Task<int> ProcessStatus(IConsole console)
         {
+            StringBuilder MakeFixedWidth(string text, int width, StringBuilder sb, bool alignRight_leftPadding = false, bool alignLeft_rightPadding = false, bool backwards = false)
```

Comment:
Nit: `alignRight_leftPadding` would be clearer as `leftPad` and same for the right padding. Align right and left pad mean the same thing in my mind